tests/test_text_generation_example.py

```diff
@@ -134,6 +150,11 @@ def _test_text_generation(
             "--reuse_cache",
             "--trim_logits",
         ]
+        if "Llama-2" in model_name:
+            command += [
+                "--attn_softmax_bf16",
```

Comment:
This could be added for all Llama models and configs (i.e. not only fp8) no?

Reply:
Actually I see that it already added on the latest main branch.
https://github.com/huggingface/optimum-habana/blob/main/tests/test_text_generation_example.py#L114
Let me remove this line.